src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -192,7 +267,7 @@ bool
 AmendmentTableImpl::unVeto (uint256 const& amendment)
 {
     ScopedLockType sl (mLock);
-    AmendmentState* s = getCreate (amendment, false);
+    AmendmentState* s = getExisting (amendment);
```

Comment:
FWIW, Nik and I agree to disagree on whether `auto` is preferable.  (Yes, I'm swimming against the tide on this one.)  I like being able to look at the source code and know that, in this particular case, I have a raw pointer in my hands.  That information is hidden by `auto`.  And my IDE doesn't tell me.  Nik has more seniority, though ;-).

I'm not always against `auto`.  It's good in generic situations or if the name of a type is just too ugly to easily read (many iterator names).  Those don't match this case.